src/utility-types.ts

```diff
@@ -24,6 +24,25 @@ export type $Keys<T extends object> = keyof T;
  */
 export type $Values<T extends object> = T[keyof T];
 
+/**
```

Comment:
All related $ValuesType code and tests should be removed

Reply:
Done.